src/sidebars/star/star-controller.js

```diff
@@ -15,33 +15,35 @@
             this.selectionService = selectionService;
 
             this.starHovered = false;
-            this.check = false;
+
+            this.stock = $scope.stock;
```

Comment:
I'm not certain I understand what you mean here. Are you saying that because `$scope` is only used in the contructor, it doesn't need to be assigned to `this.$scope`?